doc/ref_model/chapters/appendix-a.md

```diff
@@ -66,6 +68,18 @@ Without arguing for or against SR-IOV, CNTT provides the following anecdotes whi
 
 **CPU Instruction Sets and Extensions:**
 <content needed>
+  
+**Raw Disk Device Mapping**
+<content needed>
+Overview of technology:
+
+It is possible to map SAN LUNs or local disks to a VM, giving direct hardware access to the specified device.  Given the state of today's storage technology, perhaps CNTT should not provide an exception for this.
+
+**Remote Direct Memory Access**
+<content needed>
+Overview of technology:
+
+RDMA allows direct memory access from the memory of one computer to the memory of another computer without involving the operating system or CPU . The transfer of memory is offloaded to the RDMA-capable Host Channel Adapters (HCA) . A PVRDMA network adapter provides remote direct memory access in a virtual environment.
```

Comment:
If we're going to give an example of how this works in a virtual environment, we should also include the example for bare metal / containerised (or state that this is not applicable).

Reply:
I am not familiar enough with the current state of container networking to know if RDMA is available to it.  RDMA by itself is the bare metal technology, and what I mean to indicate here is there there needs to be a translation layer for the virtualised/containerised environment, and the software running must have a 'driver' or something to talk over the RDMA channel.